src/components/configMaps/ConfigMap.tsx

```diff
@@ -383,7 +411,34 @@ export function ConfigMapForm({ id, appId, name = "", external, data = null, typ
                     error={volumeMountPath.error}
                     onChange={e => setVolumeMountPath({ value: e.target.value, error: "" })} />
             </div> : null}
-
+            {selectedTab === 'Data Volume' ?
+                <div className="mb-16">
+                    <Checkbox isChecked={isSubPathChecked}
+                        onClick={(e) => { e.stopPropagation(); }}
+                        rootClassName="form__checkbox-label--ignore-cache"
+                        value={"CHECKED"}
+                        onChange={(e) => setIsSubPathChecked(!isSubPathChecked)}>
+                        <span className="mr-5"> Set subPath (Required for sharing one volume for multiple uses in a single pod)</span>
```

Comment:
this text is used at multiple places abstract out as constant

Reply:
this is not resolved rt?

Reply:
This has been resolved, extracted out  "CHECKED" as constant